code/assurance/src/main/java/com/adobe/marketing/mobile/assurance/internal/ui/AssuranceActivity.kt

```diff
@@ -26,7 +31,7 @@ import com.adobe.marketing.mobile.assurance.internal.ui.theme.AssuranceTheme.bac
 /**
  * Activity that hosts all of the Assurance UI.
  */
-internal class AssuranceActivity : ComponentActivity() {
+class AssuranceActivity : ComponentActivity() {
```

Comment:
Android requires all activities to be public. This was `internal` earlier likely by mistake.